integrationtests/src/test/java/org/axonframework/integrationtests/queryhandling/AbstractSubscriptionQueryTestSuite.java

```diff
@@ -532,21 +517,61 @@ void testBufferOverflow() {
 
         SubscriptionQueryResult<QueryResponseMessage<List<String>>, SubscriptionQueryUpdateMessage<String>> result =
                 queryBus.subscriptionQuery(queryMessage,
-                                           new SubscriptionQueryBackpressure(FluxSink.OverflowStrategy.ERROR),
```

Comment:
This change is not backwards compatible. With deprecated method an error was expected. Right now, execution is successful. We should support new and old version of API fully.